test/e2e/scheduling/priorities.go

```diff
@@ -430,17 +455,46 @@ func createBalancedPodForNodes(f *framework.Framework, cs clientset.Interface, n
 	var maxCPUFraction, maxMemFraction float64 = ratio, ratio
 	var cpuFractionMap = make(map[string]float64)
 	var memFractionMap = make(map[string]float64)
+
+	var highestUtilizedNode v1.Node
 	for _, node := range nodes {
-		cpuFraction, memFraction := computeCPUMemFraction(cs, node, requestedResource)
+		cpuFraction, memFraction, _, _ := computeCPUMemFraction(cs, node, requestedResource)
 		cpuFractionMap[node.Name] = cpuFraction
 		memFractionMap[node.Name] = memFraction
 		if cpuFraction > maxCPUFraction {
 			maxCPUFraction = cpuFraction
 		}
 		if memFraction > maxMemFraction {
```

Comment:
There's no guarantee `memFraction > maxMemFraction` condition will be true at least once. In which case `highestUtilizedNode` will have empty node name.